engine/flink/process/src/main/scala/pl/touk/nussknacker/engine/process/runner/FlinkStreamingProcessMain.scala

```diff
@@ -21,9 +22,15 @@ object FlinkStreamingProcessMain extends FlinkProcessMain[StreamExecutionEnviron
                                     processVersion: ProcessVersion,
                                     prepareExecutionConfig: ExecutionConfigPreparer): Unit = {
     val compiler = new FlinkProcessCompiler(modelData)
-    val registrar = FlinkStreamingProcessRegistrar(compiler, modelData.processConfig, prepareExecutionConfig)
+    val registrar: FlinkProcessRegistrar = prepareRegistrar(modelData, prepareExecutionConfig, compiler)
     registrar.register(env, process, processVersion)
     val preparedName = modelData.objectNaming.prepareName(process.id, modelData.processConfig, new NamingContext(FlinkUsageKey))
     env.execute(preparedName)
   }
+
+  protected def prepareRegistrar(modelData: ModelData,
```

Comment:
Do we realy need this oneline method?

Reply:
You're right, method above is small enough...